modules/installation-methods-for-different-architectures.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-installation-platforms.adoc
+:_content-type: REFERENCE
+[id="installation-methods-for-different-architectures_{context}"]
+= Installation methods for different architectures
+
+[role="_abstract"]
+{product-title} ({product-title-short}) and {product-title-managed} ({product-title-managed-short}) are supported on specific architectures.
+
+.Supported architectures and recommended installation methods
+[%autowidth]
+|===
+|*Supported architectures*|*Supported for Central*|*Supported for Secured Clusters*|*Supported installation methods*
+
+|x86-64 
+|Yes
+|Yes
+a|Operator (preferred), Helm charts, or roxctl CLI (not recommended)
+
+|IBM Power (ppc64le) 
+|No
+|Yes ({osp} versions 4.10, 4.11, and 4.12 only)
+.2+a|Operator is the only supported install method.
+
+|IBM Z and LinuxONE (s390x)
```

Comment:
**nitpick:** We must remove LinuxONE from here. After the recent discussions with team we don't want to mention this term for now.